app/views/admin/settings/index.html.erb

```diff
@@ -64,12 +64,23 @@
 
       <dt><%= f.label :cache_expiration, t(".fields.cache_expiration") %></dt>
       <dd><%= admin_setting_text_field(f, :cache_expiration, size: "3") %></dd>
+    </dl>
+  </fieldset>
 
+  <fieldset>
+    <legend><%= t(".legend.content") %></legend>
+    <dl>
       <dt><%= admin_setting_checkbox(f, :hide_spam) %></dt>
       <dd><%= f.label :hide_spam, t(".fields.hide_spam") %></dd>
 
       <dt><%= admin_setting_checkbox(f, :guest_comments_off) %></dt>
       <dd><%= f.label :guest_comments_off, t(".fields.guest_comments_off") %></dd>
+
+      <dt><%= f.label :account_age_threshold_for_comment_spam_check, t(".fields.account_age_threshold_for_comment_spam_check") %></dt>
+      <dd>
+        <%= admin_setting_text_field(f, :account_age_threshold_for_comment_spam_check, size: "3", "aria-labelledby" => "account_age_threshold_for_comment_spam_check_note") %>
+        <p id="account_age_threshold_for_comment_spam_check_note" role="label" class="footnote"><%= t(".fields.account_age_threshold_for_comment_spam_check_note") %></p>
```

Comment:
This should use `aria-describedby` because it's intended to add information rather than override the label. ["aria-label, aria-labelledby, and aria-describedby: What's the Difference?"](https://benmyers.dev/blog/aria-labels-and-descriptions/) gives a good overview of when each attribute should be used.

We also use `kebab-case` rather than `snake_case` when naming these. (The main exception is the site header, which should probably be updated to match the others at some point.)

If you'd like to see some examples of how we usually do it, you can check out [these search results for "aria-describedby"](https://github.com/search?q=repo%3Aotwcode%2Fotwarchive%20aria-describedby&type=code).